src/searcher.rs

```diff
@@ -98,12 +98,13 @@ impl Searcher {
     /// Returns `SearchResult` object.
     ///
     /// Raises a ValueError if there was an error with the search.
-    #[args(limit = 10, count = true)]
+    #[args(limit = 10, offset = 0, count = true)]
     fn search(
         &self,
         _py: Python,
         query: &Query,
         limit: usize,
+        offset: usize,
```

Comment:
Changing the order of the fields here is a breaking change, any reason why we don't want to put it at least after count?

Reply:
Got it!